argument_parser.py

```diff
@@ -6,6 +6,10 @@
 PERIOD_ARG = "-fp"
 AVALAIBLE_PERIODS = ['24h', '48h', '72h', '7days', '10days', '14days']
 DATE_ARG = "-date"
+TEMP_ARG = "-temp"
+AVALAIBLE_TEMP = ['C', 'F']
+WEATHER_ARG = '-weather_components'
+AVALAIBLE_WEATHER = ['wind', 'pressure']
```

Comment:
Only `wind` and  `pressure`? We also have at least `temperature` and `rain` mentioned in the Feature Document.

Reply:
@KrzysiekJodlowski Yes, but temperature and rain are always shown, you cannot "unchose" them. However, I agree that the list's name might be confusing so I can change it to something like ADDITIONAL_WEATHER_PARAM.

Reply:
@deerhunter4 got it, yes you are right, `temperature` and `rain` will always be considered, according to the FD, sorry I missed that somehow. And thank you for updating the variable name, I'd only suggest naming it `ADDITIONAL_WEATHER_PARAMS` since there is more than one of them.

Reply:
@KrzysiekJodlowski not up to date, check belowe.